xc/xc7/tests/ddr/README.md

```diff
@@ -0,0 +1,73 @@
+# UART-DDR test
+
+The `test_sdram.py` script in the `scripts` directory performs a calibration step with the `DDR` and calculates what are the correct bitslip and delay values to be assigned.
+
+### Prerequisites
+
+To be able to perform the test, make sure to clone the `litex` repository, as it contains the `RemoteClient` library to be able to correctly run the test.
+If you do not have the litex dependency on your machine, do the following:
+
+```
+git clone https://github.com/enjoy-digital/litex.git
```

Comment:
Does it need to be cloned into the local directory `xc/xc7/tests/ddr/`, or anywhere (like `/tmp`)?